README.md

```diff
@@ -209,7 +223,7 @@ Here are some features I'd like to eventually add to the project:
     Allows for use cases where inputs as strings are used directly (I am mostly thinking of functional testing frameworks
     like Fitnesse (fixtures) or Cucumber)
  * Support for specifying custom handlers for builder methods. Could allow: `aPerson().named("Joe").aged(45).build()`
- * Configuration of the non-standard builder methods through annotations. Something like:
+ * Support conversion in `Sets` annotation. Something like:
```

Comment:
I was not sure if that would still be a wanted feature. I only reword it as I only implemented the `@Sets` annotation.
I can remove this if not needed anymore